tests/rest/client/utils.py

```diff
@@ -54,9 +55,9 @@ class RestHelper:
     REST action, which isn't the focus of the test.
     """
 
-    hs = attr.ib()
+    hs = attr.ib(type=HomeServer)
     site = attr.ib(type=Site)
-    auth_user_id = attr.ib()
```

Comment:
We generally prefer using `auto_attribs=True` if possible, which allows us to use a slightly neater syntax in the class body. See for example

https://github.com/matrix-org/synapse/blob/5a6911598ad2d3dea96b9f8c1cffccd4f4840bf7/synapse/storage/databases/main/events_worker.py#L107-L110